demo/vagrant/Vagrantfile

```diff
@@ -74,7 +74,7 @@ Vagrant.configure(2) do |config|
   config.vm.provision "shell", inline: $script, privileged: false
 
   # Expose the nomad api and ui to the host
-  config.vm.network "forwarded_port", guest: 4646, host: 4646, auto_correct: true
+  config.vm.network "forwarded_port", guest: 4646, host: 4646, auto_correct: true, host_ip: "127.0.0.1"
```

Comment:
the change looks fine.  But I've used port forwarding with the top-level Vagrantfile without any problems - do you know what necessities it in your case?

Reply:
I tried it again without this parameter. It seems that `host_ip:` is not needed but I need to bind nomad to `0.0.0.0` by `-bind` parameter. Then I can access it from localhost.

```
nomad agent -dev -bind 0.0.0.0
```

Reply:
@pulecp I this script to run the dev agent in Vagrant because then I can make sure all the params I want are properly set, I can pass in args like log levels on the fly, and I know it's always binding to the specific IP I'm exposing.

```bash
#!/bin/bash
# starts the Nomad agent listening on the Vagrant box's
# host-accessible interface. runs as sudo so that Connect
# works properly.

sudo nomad agent \
     -dev -dev-connect \
     -bind '{{ GetInterfaceIP "eth0" }}' \
     $@
```